libevm/EVMC.cpp

```diff
@@ -14,7 +14,7 @@ namespace
 {
 evmc_revision toRevision(EVMSchedule const& _schedule) noexcept
 {
-    if (_schedule.accountVersion == IstanbulSchedule.accountVersion)
+    if (_schedule.txDataNonZeroGas == 16)
```

Comment:
Probably better to change this to `.haveChainID` after https://github.com/ethereum/aleth/pull/5696 is merged